pallets/compounding-interest/src/tests.rs

```diff
@@ -76,7 +76,9 @@ impl ExtBuilder {
 		let storage = system::GenesisConfig::default()
 			.build_storage::<TestRuntime>()
 			.unwrap();
-		TestExternalities::from(storage)
+		let mut ext = TestExternalities::from(storage);
+		ext.execute_with(|| System::set_block_number(1));
```

Comment:
Is it necessary or a good idea to set block number to 1 in so many places like this? I don't see it in the main Substrate repo.

Reply:
I checked out [Substrate v2.0.0-alpha.6](https://github.com/paritytech/substrate/tree/v2.0.0-alpha.6) and saw quite a lot of `set_block_number(1)` in the test code.

![Screenshot 2020-04-22 at 20 15 06](https://user-images.githubusercontent.com/898091/79980844-4b55bf80-84d6-11ea-8f65-7b2097da2e27.png)

But I do agree this is more like our idiosyncrasy that now Substrate developers need to be aware of when testing events...

Reply:
Oh you're right. Somehow I missed that. Thanks :+1: